recipes/flake8-class-newline/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flake8-class-newline" %}
+{% set version = "1.6.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Remove this line. This should only be set if python_min should be above the global min.
```suggestion
```